spec/sidekiq-scheduler/job_presenter_spec.rb

```diff
@@ -1,116 +1,99 @@
-require "sidekiq-scheduler/job_presenter"
+require 'sidekiq-scheduler/job_presenter'
 
 describe SidekiqScheduler::JobPresenter do
-  let(:job_name) { "job_name" }
-  let(:job_config) { { 'cron' => '* * * * *', 'class' => 'SomeIvarJob', 'args' => '/tmp' } }
+  subject(:job_presenter) { described_class.new(job_name, attributes) }
+  let(:job_name) { 'job_name' }
   let(:attributes) { {} }
 
-  subject { described_class.new(job_name, attributes) }
   before { Sidekiq.redis(&:flushall) }
 
-  describe "#next_time" do
-    context "when the job has not a next time in redis" do
-      it "returns nothing" do
-        expect(subject.next_time).not_to be
-      end
+  describe '#next_time' do
+    subject { job_presenter.next_time }
+
+    context "when the job doesn't have a next time in redis" do
```

Comment:
That's an option. Another option is to set the appropriate context using a `before` block